packages/cli/test/unit/options/beaconNodeOptions.test.ts

```diff
@@ -220,7 +220,7 @@ describe("options / beaconNodeOptions", () => {
       },
       sync: {
         isSingleNode: true,
-        slotImportTolerance: 32,
+        slotImportTolerance: 8,
```

Comment:
Why did this value change?  Seems unrelated to the runner changing...

Reply:
Because now all unit tests runs with `minimal` preset.